openshift/WFLY-15421_OIDC_for_WildFly_s2i_v2.adoc

```diff
@@ -0,0 +1,297 @@
+|SSO_ENABLE_CORS
+|Enable CORS
+|N
+|False by default 
+
+|OIDC_SECURE_DEPLOYMENT_BEARER_ONLY
+|SSO_BEARER_ONLY
+|Deployment that accepts Bearer token only, no logging.
+|N
+|False by default
+
+|OIDC_PROVIDER_SSL_REQUIRED
+|NONE, new env variable (see CLOUD-3529 )
+|Is https enabled in the realm. By default external (private and local address no https).
+|N
+|external
+
+|OIDC_PROVIDER_TRUSTSTORE
+|SSO_TRUSTSTORE
+|Name of the realm trustore file. If not set, disable-trust-manager=true
+|N
+|
+
+|OIDC_PROVIDER_TRUSTSTORE_DIR
+|SSO_TRUSTSTORE_DIR
+|Directory to find the realm truststore. If not set, disable-trust-manager=true
+|N
+|
+
+|OIDC_PROVIDER_TRUSTSTORE_PASSWORD
+|SSO_TRUSTSTORE_PASSWORD
+|Realm Truststore password. If not set, disable-trust-manager=true
+|N
+|
+
+|OIDC_PROVIDER_TRUSTSTORE_CERTIFICATE_ALIAS
+|SSO_TRUSTSTORE_CERTIFICATE_ALIAS
+|Realm trustore alias, required to interact with auth server to register client
+|N
+|
+
+|OIDC_DISABLE_SSL_CERTIFICATE_VALIDATION
+|SSO_DISABLE_SSL_CERTIFICATE_VALIDATION
+|Disable certificate validation when interacting with auth server to register client.
+|N
+|
+
+|OIDC_HOSTNAME_HTTP
+|HOSTNAME_HTTP
+|Hostname used for unsecure routes 
+|N
+|Routes are discovered
+
+|OIDC_HOSTNAME_HTTPS
+|HOSTNAME_HTTPS
+|Hostname used for secured routes
+|N
+|Secured routes are discovered
+
+|NONE
+|SSO_PUBLIC_KEY
+|Realm public key. This option is no more used, public key being automatically retrieved by the OIDC subsystem.
+|N
+|If set a warning is displayed to advertise that this option is being ignored.
+
+|===
+
+
+===== Dynamic registration of clients
+
+This registration is enabled only for the `keycloak` provider. Other providers would need the client to be already 
+registered inside the authorization server.
+
+===== Routes discovery
+
+* Route discovery (if no OIDC_HOSTNAME_HTTP(S) is set) is bound to openshift. 
+* This feature can’t be used on other cloud system.
+* The route discovery  implies that complex JSON structure be parsed.  `jq` (small runtime to parse json returned by openshift service)  is expected to be 
+installed in the WildFly s2i v2 image.
+
+==== Implementation notes
+
+* This support is implemented by a set of new bash scripts.
+* The support sources are located in the https://github.com/wildfly/wildfly-cekit-modules[wildfly-cekit-module] repository.
+* The scripts are packaged inside the `org.wildfly:wildfly-cloud-galleon-pack` Galleon feature-pack.
+* The scripts are made available when provisioning the `org.wildfly:wildfly-cloud-galleon-pack`.
+
+==== Impact on quickstarts 
+
+* NONE
+
+=== Nice-to-Have Requirements
+
+* NONE
+
+=== Non-Requirements
+
+* No support for SAML and OIDC keycloak adapters.
+
+== Test Plan
+
+* Add new Behave tests 
+* Add new Openshift QE tests.
+
+== Community Documentation
+
+* WildFly Openshift community doc is required.
+
+== Release Note Content
+
+Yes. Could be integrated in the release notes  that cover WildFly s2i v2.
```

Comment:
This section should actually contain the content of the release note that we can copy/paste :)